pdns/test-signers.cc

```diff
@@ -85,6 +86,18 @@ static const SignerParams rsaSha256SignerParams = SignerParams{
   .algorithm = DNSSECKeeper::RSASHA256,
   .isDeterministic = true,
 
+#if OPENSSL_VERSION_MAJOR >= 3
```

Comment:
Some comment about OpenSSL >= 3.0.0 using a generic key interface and thus storing the key PKCS#8-encoded might be nice?